sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeActionsOperation.cs

```diff
@@ -18,7 +17,7 @@ namespace Azure.AI.TextAnalytics
     public class AnalyzeActionsOperation : PageableOperation<AnalyzeActionsResult>, IOperation<AsyncPageable<AnalyzeActionsResult>>
     {
         /// <summary>Provides communication with the Text Analytics Azure Cognitive Service through its REST API.</summary>
-        //private readonly TextAnalyticsRestClient _serviceClient;
+        private readonly ServiceClient _serviceClient;
```

Comment:
Not sure if this pattern will ultimately work out. I shifted it for now, as it gives us a neutral client surface. The previous implementation was commented out, so this has no functionality impact just yet.

Reply:
hmm interesting. I forgot that LROs are like little mini clients. Def need to think about this one more

Reply:
Yeah.... this was a surprise that'll need a bit more design.

Reply:
Added comment to issue https://github.com/Azure/azure-sdk-for-net/issues/28330